include/glow/Backends/BackendUtils.h

```diff
@@ -29,13 +29,19 @@ struct RuntimeSymbolInfo {
   size_t offset;
   /// Type of symbol.
   Type type;
+  /// Is the symbol an input for the function.
+  bool input;
+  /// Is the symbol an output for the function.
+  bool output;
 };
 
+using SymbolTable = std::unordered_map<std::string, RuntimeSymbolInfo>;
```

Comment:
Do we need to update or read the symbol table at inference time? Updating and accessing the symbol table can be expensive (malloc for update, hash for lookup).